pyriemann/classification.py

```diff
@@ -281,6 +280,7 @@ def fit(self, X, y):
         self : FgMDM instance
             The FgMDM instance.
         """
+        self.classes_ = numpy.unique(y)
```

Comment:
it would be cleaner to just do `self.classes_ = self._mdm.classes_` after the fit. this way we are sure there isn't any discrepancy